manifest.json

```
@@ -1,25 +1,20 @@
 {
-    "name": "Abstract Bot",
+    "manifest_version": 2,
```

@Sgi114 
Chrome拡張 V2は、2023年1月からオワコンです。
V3を使用してください！
参考：https://support.google.com/chrome/a/thread/175260822?hl=en

```suggestion
    "manifest_version": 3,
```

---

@Sgi114 ✅ OK
https://github.com/Sgi114/Summarizer/pull/1/commits/a31201eb895065a44b9da4e8f5e6bb6a446b40aa#diff-ffa5b716b5a57837f7929dfcca4b4dfdeb97210a7fd5a12d2f1978846d6f1743R2